src/redux/admin/admin.reducer.js

```diff
@@ -0,0 +1,46 @@
+import {
```

Comment:
Please, rename all folder in redux to `auth` and all of the files too. It is actually authentication-related functionality and it will be much more convenient to name it `auth`

Reply:
You're right! Changed!